### PR TITLE
Update dependencies for 5.5.0

### PR DESCRIPTION
**Pull request by str4d**

### CODE REVIEW

**Comment by str4d** [comment]
I added ISRG as another source of Rust dependency audits. Combined with our existing sources, along with my own audits, here is the improvement from this PR:
- Before: 42 fully audited, 70 partially audited, 137 exempted
- After: 55 fully audited, 84 partially audited, 107 exempted

**Comment by r3ld3v** [comment]
@str4d how do you add ISRG as another source?

**Comment by r3ld3v** [comment]
> 

Ah, found it:

https://github.com/zcash/zcash/pull/6443/commits/fe05310e37dbe827fc1c6744c7bfa95e266ba569

File: qa/supply-chain/imports.lock [hunk 1]
```diff
@@ -21,41 +19,29 @@ Well documented invariants, good assertions for those invariants in unsafe code,
...
 who = "Alex Crichton <alex@alexcrichton.com>"
 criteria = "safe-to-deploy"
 version = "1.0.0"
 notes = "I am the author of this crate."
 
+[[audits.bytecode-alliance.audits.constant_time_eq]]
+who = "Nick Fitzgerald <fitzgen@gmail.com>"
+criteria = "safe-to-deploy"
+version = "0.2.4"
+notes = "A few tiny blocks of `unsafe` but each of them is very obviously correct."
```
Comment (daira) [inline qa/supply-chain/imports.lock:43]: I wouldn't say it's obvious; the unsafe `asm!` blocks obviously don't *introduce* unsoundness, but it was less clear to me that they inhibit optimizations as intended. I filed https://github.com/cesarb/constant_time_eq/issues/8 and my questions about that code were satisfactorily answered.

**Review by daira — APPROVED** [review]
utACK

**Pull request merged**